src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box } from '@material-ui/core';
+import React from 'react';
+
+import { useGetContactDetailsHeaderQuery } from './ContactDetailsHeader.generated';
+
+interface Props {
+  accountListId: string;
+  contactId: string;
+}
+
+export const ContactDetailsHeader: React.FC<Props> = ({
+  accountListId,
+  contactId,
+}: Props) => {
+  const { data, loading } = useGetContactDetailsHeaderQuery({
+    variables: { accountListId, contactId },
```

Comment:
What should we do about errors for this component? It would be great if we could have a consistent pattern for handling and displaying errors. I've used ``` const { enqueueSnackbar } = useSnackbar();``` and it seems to work really well but not sure if everyone else is on board with it.

Reply:
I'm good with Snackbar, we should discuss if we're good with keeping that or if we should find something else.

Reply:
Probably a merged PR isn't the best place for this discussion but sticking it here anyway...

I think we could handle errors globally with an Apollo link like https://github.com/CruGlobal/missionhub-react-native/blob/develop/src/apolloClient.ts#L27-L57.

Since we're using the snackbars which just display over everything and we aren't displaying anything at the component level, we probably don't need to make the individual components do anything.

Guess we should make a Jira ticket if we want to go this route.

Or maybe design has thoughts... The UX could be better if we put the error message right where it occurred. But it's probably more work to do that on a per component basis.

What should the component do if it gets an error and no data? Try and stay in the loading state? Render nothing? I guess we'll already have to be handling any fields that aren't defined.